### DISCUSSION TITLE
Welcome to spiceoffabric Discussions!

### DISCUSSION THREAD

**Discussion by Siphalor** [Siphalor/spiceoffabric#64] [Announcements]
## 👋 Welcome!

I decided to set up Discussions as a way to get general feedback or answer questions aside from the Discord, as not everyone is comfortable with that.

This is also a good place to share your configurations and discuss issues with them.

**Comment by Jaylorod** [comment]
I have been running a server with this mod as a cure for multiple origins' low health without completely overriding it. I would like to keep carrot mode but many of us are dying over the restrictiveness of the main mod. I know the way I want the config to look, but after changing it in the files nothing changes on the server. What do I do?
